### ISSUE TITLE
Milestone 2 Deliverables

### ISSUE THREAD

**Issue by gina-gerlach** [gina-gerlach/dsta-2025-1#11]
**Deliverables:**

- [ ] Create a release (as in the last report) for the commit in the master/main branch. The tag should be called "milestone_2". 
- [x] This release must contain your "Dockerized" code and your report. We will essentially just "docker build/run" your code on the server. It has to work like this. 
- [x] With “work” we mean that the code should download the necessary training files, train a neural network, save the trained neural network as a “.h5” file, load the “.h5” file and then perform inference with the loaded neural network on a not previously seen image (test set). 
- [x] We would like to see a print of the inference result of the neural network.
- [x] Docker should deal with all the dependencies necessary to run your code.
